var/spack/repos/builtin/packages/py-tns/package.py

```diff
@@ -22,8 +23,8 @@ class PyTns(PythonPackage):
     depends_on('py-setuptools', type='build')
 
     depends_on('py-matplotlib@1.3:', type='run')
-    depends_on('py-morphio@2.3.10:', type='run')
-    depends_on('py-neurom@1.4.19:', type='run')
+    depends_on('py-morphio@3.1.1:', type='run')
+    depends_on('py-neurom@2.1.2:', type='run')
```

Comment:
Some dependencies are different from `setup.py`, that in `2.4.2` defines for `install_requires`:
```
        'matplotlib>=1.3.1',
        'tmd>=2.0.8',
        'morphio>=2.7.1',
        'neurom>=2,<3',
        'scipy>=0.13.3',
        'numpy>=1.15.0',
        'jsonschema>=3.0.1',
```